mizar/proto/mizar/proto/builtins.proto

```diff
@@ -68,6 +73,12 @@ message BuiltinsArktosMessage {
   string vpc = 2;
 }
 
+message BuiltinsNetworkPolicyMessage {
+  string name = 1;
+  string namespace = 2;
+  string tenant = 3;
```

Comment:
Whether network policy has tenant field?

Reply:
Yes, it should